listenbrainz/webserver/static/js/src/utils/APIService.ts

```diff
@@ -363,13 +363,19 @@ export default class APIService {
   };
 
   getUserEntity = async (
-    userName: string,
+    userName: string | undefined,
```

Comment:
Our convention in other files is to denote optional props like this (whihc ends up as string | undefined):
```suggestion
    userName?: string,
```

Reply:
Right, I had seen that at a few places and used it earlier but I got an error because its followed by entity which is not optional. So I had gone with this format everywhere. Changing all other instance in the PR except this one should be fine will do.

Reply:
Fixed.